api_utils/txtlib.py

```
@@ -0,0 +1,131 @@
+            # ---
+            # print("te:%s, namestrip:%s" % (te,namestrip) )
+            # ---
+            tabe = {
+                namestrip: params
+            }
+            named.append(tabe)
+    # ---
+    return named
+
+
+def get_all_temps_params(text, templates=[], lowers=False):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)